Frontend/src/components/globalStyles/Loader.jsx

```diff
@@ -1,7 +1,6 @@
-import React, { useState } from 'react';
-import styles from './Loader.css';
```

Comment:
I don't think the loader styles were to be deleted

Reply:
you are right. added it back